.changeset/stupid-news-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Separator` - Convert component to TypeScript
```

Comment:
We seem to be aiming for past tense

```suggestion
`Separator` - Converted component to TypeScript
```

Reply:
👍  made this change on my initial commit